server/controllers/events/events_controller_e2e_test.go

```diff
@@ -1016,16 +1011,14 @@ func setupE2E(t *testing.T, repoDir, repoConfigFile string) (events_controllers.
 
 	Ok(t, err)
 
-	conftestVersion, _ := version.NewVersion(ConftestVersion)
-
 	conftextExec := policy.NewConfTestExecutorWorkflow(logger, binDir, &NoopTFDownloader{})
 
 	// swapping out version cache to something that always returns local contest
 	// binary
 	conftextExec.VersionCache = &LocalConftestCache{}
 
 	policyCheckRunner, err := runtime.NewPolicyCheckStepRunner(
-		conftestVersion,
+		defaultTFVersion,
```

Comment:
asis impl is invalid. `runtime.NewPolicyCheckStepRunner` first parameter is defaultTfVersion.